test/runtime/Policies.go

```diff
@@ -532,67 +539,6 @@ var _ = Describe("RuntimeValidatedPolicies", func() {
 
 		connectivityTest(allRequests, helpers.App1, helpers.Httpd1, true)
 		connectivityTest(allRequests, helpers.App2, helpers.Httpd1, true)
-
-		By("Ingress CIDR")
```

Comment:
I agree that these CIDR tests were not testing the actual CIDR policies.

If there's not already an issue for reviewing the CIDR policy coverage, we should file one.